lib/streamlit/stats.py

```diff
@@ -92,13 +91,14 @@ def options(self):
         self.finish()
 
     def get(self) -> None:
-        metric_type = "# TYPE st_cache_memory_bytes gauge"
-        metric_unit = "# UNIT st_cache_memory_bytes bytes"
+        metric_type = "# TYPE cache_memory_bytes gauge"
+        metric_unit = "# UNIT cache_memory_bytes bytes"
         metric_help = "# HELP Total memory consumed by a cache."
         openmetrics_eof = "# EOF\n"
-        stats = [stat.to_metric_str() for stat in self._manager.get_stats()]
-        foo = [metric_type, metric_unit, metric_help]
-        foo.extend(stats)
-        foo.append(openmetrics_eof)
-        self.write("\n".join(foo))
+
+        # Format: header, stats, EOF
+        stats = [metric_type, metric_unit, metric_help]
+        stats.extend(stat.to_metric_str() for stat in self._manager.get_stats())
+        stats.append(openmetrics_eof)
+        self.write("\n".join(stats))
```

Comment:
Shuffled around to remove presumably placeholder "foo" var name

Reply:
haha, whoops, yep